frontend/src/components/dashboard/aliases/AliasDeletionButton.module.scss

```diff
@@ -20,7 +20,8 @@
   @include text-body-md;
 
   &:hover {
-    color: $color-red-70;
+    background-color: $color-red-70;
```

Comment:
Heads-up: Adding a proper hover state for the deletion button is the only change that's not behind a flag.